MOBILE_API.xml

```diff
@@ -364,6 +364,10 @@
     </enum>
 
     <enum name="PredefinedWindows" since="6.0">
+    	<description>
+    		Specifies what windows and IDs are predefined and pre-created. 
+    		Mobile libraries and core communicate using the integer value.
+    	</description>
         <element name="DEFAULT_WINDOW" value="0">
             <description>The default window is a main window pre-created on behalf of the app.</description>
         </element>
```

Comment:
```suggestion
    <enum name="PredefinedWindows" since="6.0">
        <description>
            Specifies IDs for windows are predefined and automatically created.
            The mobile libraries and SDL Core use the integer value when referencing these elements.
        </description>
        <element name="DEFAULT_WINDOW" value="0">
            <description>The default window is a main window pre-created on behalf of the app.</description>
        </element>
```